src/rules/no-export.js

```diff
@@ -0,0 +1,49 @@
+import { getDocsUrl, isTestCase } from './util';
+
+let exportNodes = [];
```

Comment:
These should be inside the `create` to not leak between tests

Reply:
done, thx